packages/react/package.json

```diff
@@ -18,21 +18,27 @@
     "report": "WITH_REPORT=true pnpm build"
   },
   "dependencies": {
-    "@logto/browser": "^0.1.0"
+    "@logto/browser": "^0.1.0",
+    "@silverhand/essentials": "^1.1.6"
   },
   "devDependencies": {
     "@jest/types": "^27.5.1",
+    "react": "^17.0.2",
     "@silverhand/eslint-config": "^0.9.1",
     "@silverhand/eslint-config-react": "^0.9.1",
     "@silverhand/ts-config": "^0.9.1",
     "@silverhand/ts-config-react": "^0.9.1",
+    "@testing-library/react-hooks": "^7.0.2",
     "@types/jest": "^27.4.1",
+    "@types/react": "^17.0.39",
+    "codecov": "^3.8.3",
```

Comment:
do we need this now?

Reply:
can b removed using github actions instead

Reply:
Removed.